interface/frontend/src/views/DetectAndSelect.vue

```diff
@@ -1,5 +1,7 @@
 <template>
+<div class="offset-top">
```

Comment:
The class `offset-top` is local for `/AnnotateAndSegment.vue` and `/ReportAndExport` at the moment. Refer to this discussion: https://github.com/concept-to-clinic/concept-to-clinic/pull/173#issuecomment-337597767

We might move `offset-top` up as a project-class? @lamby

Reply:
Actually, I added it to App.vue:
https://github.com/concept-to-clinic/concept-to-clinic/pull/226/files#diff-99d7a1bed87dbda2f915d10103d13394R28
But probably, it's a good idea to remove the copy\paste from other views.